types/ws/index.d.ts

```diff
@@ -70,101 +70,106 @@ declare class WebSocket extends EventEmitter {
         options?: WebSocket.ClientOptions | ClientRequestArgs,
     );
 
-    close(code?: number, data?: string | Buffer): void;
-    ping(data?: any, mask?: boolean, cb?: (err: Error) => void): void;
-    pong(data?: any, mask?: boolean, cb?: (err: Error) => void): void;
-    send(data: any, cb?: (err?: Error) => void): void;
+    close(code?: number, data?: WebSocket.Data): void;
```

Comment:
the change to `ws.close` doesn't match [what the docs say](https://github.com/websockets/ws/blob/master/doc/ws.md#websocketclosecode-reason), but I think it might be the docs that are wrong here (?)